preamble/syllabus.qmd

```diff
@@ -7,45 +7,55 @@
 Our overall learning outcome for the course is:
 
 1.  Describe some core features of effective team-based, collaborative
-    workflows and identify the components that make these workflows
-    effective compared to other workflows. Then use Git and GitHub with
-    approaches that strongly support effective collaboration.
+    workflows and use Git and GitHub with approaches that support
+    effective collaboration.
```

Comment:
It's not that I don't think the removed part is important, I'm just not sure I think it should be a part of the overall learning outcome. I think it was a bit "wordy"/long before :)

Reply:
Sounds fine with me :relaxed: